crates/cext/src/dag.rs

```diff
@@ -736,3 +736,62 @@ pub unsafe extern "C" fn qk_dag_free(dag: *mut DAGCircuit) {
         }
     }
 }
+/// @ingroup QkDag
+/// Return the operation nodes in the DAG listed in topological order.
+///
+/// @param dag A pointer to the DAG.
+/// @param order A pointer to an array of ``qk_dag_num_op_nodes(dag)`` elements
+/// of type ``uint32_t``, where this function will write the output to.
+///
+/// # Example
+/// ```c
+/// QkDag *dag = qk_dag_new();
+/// QkQuantumRegister *qr = qk_quantum_register_new(1, "my_register");
+/// qk_dag_add_quantum_register(dag, qr);
+///
+/// uint32_t qubit[1] = {0};
+/// qk_dag_apply_gate(dag, QkGate_H, qubit, NULL, false);
+/// qk_dag_apply_gate(dag, QkGate_S, qubit, NULL, false);
+///
+/// // get the number of operation nodes
+/// uint32_t num_ops = qk_dag_num_op_nodes(dag); // 2
+/// uint32_t *order = malloc(sizeof(uint32_t) * num_ops);
+///
+/// // get operation nodes listed in topological order
+/// qk_dag_topological_op_nodes(dag, order);
+///
+/// // do something with the ordered nodes
+/// for (uint32_t i=0; i<num_ops; i++) {
```

Comment:
```suggestion
/// for (uint32_t i = 0; i < num_ops; i++) {
```
Just to have it comply with how `clang-format` would format it.